### ISSUE TITLE
Persistent storage fail

### ISSUE THREAD

**Issue by fgadaleta** [dakk/kad.py#6]
Getting keys with persistent storage fails.
Below how to replicate the error

```
from kad import DHT
import shelve

# bootstrap node
host1, port1 = 'localhost', 3000

# this node
host2, port2 = 'localhost', 3001
dht2 = DHT(host2, port2, seeds=[(host1,port1)], storage=shelve.open ('storage_two.dat'))

print (dht2["file00"])	# blocking get
dht2.get("file00", lambda d: print(d)) # threaded get
```

```Traceback (most recent call last):
  File "two.py", line 12, in <module>
    print (dht2["file00"])	# blocking get
  File "/Users/frag/Documents/kad.py/kad/kad.py", line 199, in __getitem__
    if hashed_key in self.data:
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/shelve.py", line 102, in __contains__
    return key.encode(self.keyencoding) in self.dict
AttributeError: 'int' object has no attribute 'encode'
```

**Comment by urdarinda** [comment]
@dakk  This happens to me also.

**Comment by dakk** [comment]
Thank you for reporting; atm I don't have available time to fix the problem. If one of you can, please submit a fix trough a pull request

**Comment by dakk** [comment]
Fixed by "Merge pull request #7 from romeupalos/master"

**Comment by dakk** [comment]
Released on pypi https://pypi.python.org/pypi/kad.py/0.6.0
